js/cryptocom.js

```diff
@@ -483,6 +483,77 @@ module.exports = class cryptocom extends Exchange {
         return result;
     }
 
+    async fetchLastPrices (symbols = undefined, params = {}) {
```

Comment:
@ttodua question, should we add something in the docs regarding this new method?

Reply:
yes, but after we add around 5-10 exchanges at least

Reply:
@ttodua fair enough, but shouldn't we add it at least to `has` inside Exchange.js?

Reply:
@carlosmiei done! please re-review now.

Reply:
@carlosmiei can you take a look again in this?

Reply:
@ttodua sure, will take a look soon